### PR TITLE
URL extraction documentation

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
Ready

## Description
An article for the new URL extraction flow and components.

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@altmannyarden Doc review completed.

**Comment by altmannyarden** [comment]
@ytauman FYI

File: docs/indicators/url-extraction.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+id: url-extraction
+title: URL Extraction
+---
+
+
+The Cortex XSOAR URL indicator type is built using regex and a formatting script.
+The following describes the URL indicator components and what output you should expect when extracting indicators of type URL.
+
+## URL Indicator Components
```
Comment (altmannyarden) [inline docs/indicators/url-extraction.md:10]: Its the extraction components :) the URL indicator has more components such as reputation and enrichment

File: docs/indicators/url-extraction.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+The Cortex XSOAR URL indicator type is built using regex and a formatting script.
+The following describes the URL indicator components and what output you should expect when extracting indicators of type URL.
+
+## URL Indicator Components
+
+There are two components when extracting URL indicators:
+- Regex
+- Formatting script
+
+### Regex
```
Comment (altmannyarden) [inline docs/indicators/url-extraction.md:16]: regex extraction

File: docs/indicators/url-extraction.md (hunk 1 above)
Comment (melamedbn) [reply docs/indicators/url-extraction.md:10]: The URL itself is built using regex and a formatting script.
The reputation and enrichments are post-creation.

File: docs/indicators/url-extraction.md (hunk 2 above)
Comment (melamedbn) [reply docs/indicators/url-extraction.md:16]: The name of the component is Regex, the extraction is the output.
It's just the name of the two components, a detailed explanation is provided afterward.

File: docs/indicators/url-extraction.md (hunk 1 above)
Comment (altmannyarden) [reply docs/indicators/url-extraction.md:10]: true :) but they are still part of the URL component

Comment (melamedbn) [reply docs/indicators/url-extraction.md:10]: So I think it shouldn't be called 'URL Extraction' but 'URL Indicator' with the all details.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**